imports/products.go

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See ENTERPRISE-LICENSE.txt and SOURCE-CODE-LICENSE.txt for license information.
+
+package imports
+
+import (
+	// Needed to ensure the init() method in the FocalBoard product is run
+	_ "github.com/mattermost/focalboard/mattermost-plugin/product"
```

Comment:
I think we should create this only if there is focalboard project directory next to mattermost-server. Like we do for the enterprise.

Reply:
@isacikgoz  
I get it for enterprise, due to the open-core nature, but why complicate building two open source products?  All that Makefile plumbing will need to be duplicated for each product.  What is the gain?

Reply:
I see, I was thinking from a developer experience point of view actually. To run the mattermost-server we will be adding another dependency. I'm 0/5 on not adding this but would like to hear others as well. cc @jwilander

Reply:
I have started a thread on this [here](https://community.mattermost.com/core/pl/7qeoamgn3bgkdy75a684io93gy)

Reply:
Good point, thanks!

Reply:
> we should create this only if there is focalboard project directory next to mattermost-server.

Done